0xLiso/minetest-shadowmap
# Tilted shadows

This PR adds proper support for tilted orbits for Sun and Moon and, of course the generated shadows.

* New setting shadow_sky_body_orbit_tilt, value in degrees from 0.0 to 60.0, angle is from the vertical.
* Aligned light position for shadow mapping with the position of the matching body in the sky

## How to test

* Go to the settings and set shadow_sky_body_orbit_tilt to, say, 30 degrees.
* Launch the game, set /time 10:00
* Observe the Sun position and shadows
